From 7283013335933a44d0a6261d14068852878846a6 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Fri, 29 Mar 2024 11:08:32 +0100
Subject: [PATCH 0470/1141] arm64: dts: st: set max SPI NOR frequency to 84 MHz

Set max SPI NOR frequency to 84 MHz.

Change-Id: I6fb658e793ae9c4db09aba0eaa849cd5854adc1a
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/327927
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -495,7 +495,7 @@
 			reg = <0>;
 			spi-rx-bus-width = <4>;
 			spi-tx-bus-width = <4>;
-			spi-max-frequency = <133000000>;
+			spi-max-frequency = <84000000>;
 		};
 	};
 };
